### ISSUE TITLE
A11y_Azure data studio_Query History_Executed query_Screenreader: Screen reader is not announcing the any message whether it is context menu present on "query history" row for right click to open the menu. 

### ISSUE THREAD

**Issue by Jtith** [microsoft/azuredatastudio#20890]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHub Tags:
#A11yTCS; #A11ySev4; #A11yUsable; #SH_SQLAzureDataStudio_Win32_Oct2022; #SQL Azure Data Studio; #Screen reader; #NVDA; #JAWS; #Win11; #FTP; #DesktopApp; #Win32;

### Environment Details:
Application Name: SQL Azure Data Studio
Windows 11 Version: 22621.674
NVDA Version 2020.4
JAWS Version 2021.205.53

### Repro Steps: -

1. Install Azure Data Studio – Insiders
2. Tab till New connection and press enter
3. Using valid credentials, connect to server
4. Tab till extensions and press enter
5. Tab till search box and search "Query History" 
6. Tab till Install and press enter
7. Tab till new query and press enter
8. Tab till query editor and execute any query
9. Open Toggle panel using Ctrl+J
10. Turn on NVDA
11. Tab till Query History tab and press enter
12. Tab till the executed query in rows below Query history
13. Check whether the screen reader is announcing any message whether it is context menu present on "query history" row for right click to open the menu. 

### Actual:
Screen reader is not announcing the any message whether it is context menu present on "query history" row for right click to open the menu. 

### Expected:
Screen reader should announce the any message whether it is context menu present on "query history" row for right click to open the menu. 

### User Impact:
Screen reader users will not understand whether the context menu is collapsed to open.

### Attachment: -
[Screen reader is not announcing the any message whether it is context menu present on _query history.zip](https://github.com/microsoft/azuredatastudio/files/9826680/Screen.reader.is.not.announcing.the.any.message.whether.it.is.context.menu.present.on._query.history.zip)
[A11y_Azure data studio_Query History_Executed query_Screenreader.zip](https://github.com/microsoft/azuredatastudio/files/9830497/A11y_Azure.data.studio_Query.History_Executed.query_Screenreader.zip)

**Comment by Charles-Gagnon** [comment]
@Jtith What is this "any message"? I'm not aware of any requirement that an element needs to announce when it has a context menu and this isn't something we do anywhere else in the app.

Can you provide a link to the documentation explaining this and/or an example of this being done elsewhere?

**Comment by Jtith** [comment]
As per discussion sev4 bugs are not to be logged for Azure data studio - insider, hence closing this

**Comment by mstechie** [comment]
GitHubTags:#SH-SQL Azure Data Studio-Win32-OCT2022;
